modules/imgproc/src/thresh.cpp

```diff
@@ -1159,6 +1159,9 @@ getThreshVal_Otsu_8u( const Mat& _src )
 
     const int N = 256;
     int i, j, h[N] = {0};
+    #if CV_ENABLE_UNROLLED
+    int h2[3][N] = {0};
```

Comment:
Could you please change name for this to `h_unrolled`?

```
int h_unrolled[3][N] = {};
```

(`= {}` initialization should make MacOSX builder happy)